src/components/mixins/TokenPairMixin.ts

```diff
@@ -10,6 +10,7 @@ import router from '@/router'
 import { PageNames } from '@/consts'
 import { getMaxValue, isMaxButtonAvailable, isWalletConnected, isXorAccountAsset, hasInsufficientBalance } from '@/utils'
 
+// TODO: Ask is it correct name?
```

Comment:
If you have any suggestions please rename) Here it was meant that it's a function for `TokenPairMixin` creation.

Reply:
Ok, I see. Thank you, I'll leave the previous one.